src/Alert/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { forwardRef } from 'react';
+import styled from 'styled-components';
+import { MinervaProps, systemProps, Flex } from '../layout';
+import Icon from '../Icon';
+
+const StyledAlert = styled(Flex)(
+  props => ({
+    padding: '16px',
+  }),
+  systemProps
+);
+
+const StyledAlertTitle = styled('p')(
+  {
+    fontWeight: 'bold',
+    margin: '0 5px 0 8px',
+  },
+  systemProps
+);
+
+const StyledAlertDescription = styled.p``;
+
+export interface AlertProps extends MinervaProps {
+  children?: React.ReactNode;
+  status?: string;
+  title?: string;
+  description?: string;
+  color?: string;
+  icon?: string;
+  props?: any;
+}
+
+export const Alert = forwardRef(function Alert(
+  { title, description, status, color, icon, ...props }: AlertProps,
```

Comment:
Unfortunately, I think we might need a different prop name than `color`, because the color prop can also be used to change the text color. So I don't think you wouldn't be able to change the text color of this component as a user without using the `style={{}}` prop.

Reply:
I knew I couldn't call it backgroundColor but I have no idea why it didn't click that 'color' is also a prop?